src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -73,6 +73,7 @@ def create_target_image(location, transient_resource_group_name, source_type, so
 
     sas_token = run_cli_command(cli_cmd)
     sas_token = sas_token.rstrip("\n\r")  # STRANGE
+    sas_token = sas_token.split()[-1]  # Removing the warnings if there is any python lib warnings in output
     logger.debug("sas token: %s", sas_token)

```

Comment:
It is recommended not to directly modify the SAS token variable to avoid unexpected issues that may arise in the future when it is called

```suggestion
    logger.debug("sas token: %s", sas_token.split()[-1])
```

Reply:
@zhoxing-ms Changing output of run_cli_command(cli_cmd) function return can be more risky, as a lot of other things might be dependent on that.
Although that would be ideal way to trim Warnings from function return